api/src/pcapi/core/offerers/models.py

```diff
@@ -1067,7 +1067,13 @@ class ApiKey(PcObject, Base, Model):
     secret: bytes = Column(LargeBinary, nullable=True)
 
     def check_secret(self, clear_text: str) -> bool:
-        return crypto.check_password(clear_text, self.secret)
+        if self.secret.decode("utf-8").startswith("$2b$"):
+            if crypto.check_password(clear_text, self.secret) == False:
```

Comment:
[détail] plus pythonique : 

```suggestion
            if not crypto.check_password(clear_text, self.secret):
```